automation/vars/uni01alpha.yaml

```diff
@@ -0,0 +1,63 @@
+---
+vas:
+  uni01alpha:
+    stages:
+      - path: examples/dt/uni01alpha/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60s
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - pre_stage_run:
+          - name: Apply cinder-lvm label on master-0
+            type: cr
+            definition:
+              metadata:
+                labels:
+                  openstack.org/cinder-lvm: ""
+            kind: Node
+            resource_name: master-0
+            state: patched
+        path: examples/dt/uni01alpha/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackcontrolplane
+            controlplane
+            --for condition=Ready
+            --timeout=30m
+        values:
+          - name: network-values
+            src_file: nncp/values.yaml
+          - name: service-values
+            src_file: service-values.yaml
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni01alpha/networker
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackdataplanedeployment
+            networker-deploy
+            --for condition=Ready
+            --timeout=1200s
+        values:
+          - name: edpm-values
+            src_file: values.yaml
+        build_output: edpm-networker.yaml
+
+      - path: examples/dt/uni01alpha
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackdataplanedeployment
+            edpm-deployment
+            --for condition=Ready
+            --timeout=1200s
```

Comment:
timeout wise the value for this should be higher that networker as deploy more services.
If as part of CI we want to override this value how to do it?

For me it once take 21 minute and once around 30 minute.

Reply:
Okay... at my end running on the CI system, it was able to complete the end to end architecture deployment under 40 minutes

```
Thursday 18 April 2024  06:51:48 -0400 (0:00:00.182)       0:36:23.438 ******** 
=============================================================================== 
kustomize_deploy : Run Wait Conditions for examples/dt/uni01alpha ------------------------------------------------------------------------------------------------------------------------------------------------------------------- 756.00s
kustomize_deploy : Run Wait Conditions for examples/dt/uni01alpha/networker --------------------------------------------------------------------------------------------------------------------------------------------------------- 475.35s
kustomize_deploy : Run Wait Conditions for examples/dt/uni01alpha/control-plane ----------------------------------------------------------------------------------------------------------------------------------------------------- 349.54s
os_must_gather : Run openstack-must-gather command ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 207.00s
ci_setup : Install needed packages --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 54.38s
openshift_obs : Wait for observability-operator pod ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 50.59s
kustomize_deploy : Wait for cert-manager-operator pods ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 40.75s
kustomize_deploy : Wait for MetalLB speaker pods ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 30.85s
kustomize_deploy : Wait for certmanager pods ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 26.03s
artifacts : Ensure we have proper rights on the gathered content --------------------------------------------------------------------------------------------------------------------------------------------------------------------- 19.00s
kustomize_deploy : Wait for NMstate webhook deployment ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 15.76s
kustomize_deploy : Wait for NMstate handler pods ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 15.64s
repo_setup : Generate repos using rhos-release ceph-7.0-rhel-9 -r 9.3 ---------------------------------------------------------------------------------------------------------------------------------------------------------------- 13.57s
kustomize_deploy : Wait for cainjector pods ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ 10.58s
kustomize_deploy : Run Wait Conditions for examples/dt/uni01alpha/control-plane/nncp -------------------------------------------------------------------------------------------------------------------------------------------------- 6.74s
kustomize_deploy : Wait for webhook pods ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 5.52s
ci_local_storage : Fetch hostnames for all hosts -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 5.18s
ci_local_storage : Perform action in the PV directory --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 4.18s
ci_local_storage : Perform action in the PV directory --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 3.71s
kustomize_deploy : Apply generated content for examples/dt/uni01alpha/control-plane/nncp ---------------------------------------------------------------------------------------------------------------------------------------------- 3.37s
```
let me double the values just in case we are running on a slower systems.